### ISSUE TITLE
Tests failing with newer perl versions (5.27+)

### ISSUE THREAD

**Issue by simcop2387** [pjcj/Devel--Cover#189]
```
#   Failed test 'test with perl -MDevel::Cover,-silent,1 -e'                                                                                                                                 
#   at t/internal/subprocess.t line 18.                                                                                                                                                      
#          got: 'B::Debug will be removed from the Perl core distribution in the next major release. Please install it from CPAN. It is being used at                                        
+/home/ryan/.cpanm/work/1501762942.15852/Devel-Cover-1.26/blib/lib/Devel/Cover.pm, line 27.                                                                                                  
# Hello, world.' 
#     expected: 'Hello, world.'                                                                                                                                                              
# Looks like you failed 1 test of 2.
```

Adding an explicit dependency on B::Debug should fix it while still also working with older perl versions.

**Comment by pjcj** [comment]
Yep, just needs to be done.  Thanks for the report.

**Comment by pjcj** [comment]
I have put a fix in place with ad4c2ab39d42adcdf93ba2551c6f89efb20a348d.  Thanks very much for your report.

I have also added you to the contributors file.  Please feel free to adjust your entry as you see fit.

Thanks again.
